### ISSUE TITLE
[Model/Framework] Bert base Nan loss

### ISSUE THREAD

**Issue by Esaada** [NVIDIA/DeepLearningExamples#570]
Hi, I'm trying to pretrain Bert base, I followed the quick start, everything went smoothly,
Now I want to training using this command line:
` bash scripts/run_pretraining_lamb.sh 64 8 8 1e-4 5e-5 fp32 false 8 2000 200 7820 100 512 1024 base 2>&1 | tee -a reference_training.log &`
I've tried several attempts, as you can see the learning rate is smaller than for bert large command lines, and I increased the gradient

 accumulation, although it still didn't helped, and I'm still getting Nan loss.

I'm working on 8 V100 32GB gpus.

I'm attaching the full log file.
[reference_training.txt](https://github.com/NVIDIA/DeepLearningExamples/files/4809076/reference_training.txt)
Thanks.

**Comment by swethmandava** [comment]
Could you try to enable fp16 and XLA? 

We have tried pretraining base in the context of [biobert](https://github.com/NVIDIA/DeepLearningExamples/blob/master/TensorFlow/LanguageModeling/BERT/biobert/scripts/run_pretraining_pubmed_base_phase_1.sh) if that helps.

**Comment by Aurelius84** [comment]
I got the same problem while training bert base on V100 16G with `num_gpu=8`. The pre-training job will exit because loss is Nan.

My command line :
```
bash scripts/run_pretraining_lamb.sh  48 64 8 7.5e-4 5e-4 $precision true 82000 200 200 200 88 512 base
```

**Comment by swethmandava** [comment]
I was able to use the same recipe (same global batch size, learning rate and steps. however I am able to fit a bigger bs/gpu for bert base) as bert large for bert base and converge.
![bert_base_pretraining](https://user-images.githubusercontent.com/17828952/108259978-994b4c00-7116-11eb-92d3-188a844133a6.png)

@Aurelius84 i see you modified the number of training/warmup steps and gbs. Could you try using our bert large recipe instead?
